integration-cli/docker_cli_info_unix_test.go

```diff
@@ -3,15 +3,30 @@
 package main
 
 import (
-	"strings"
+	"context"
 	"testing"
 
+	"github.com/docker/docker/client"
 	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
 )
 
 func (s *DockerSuite) TestInfoSecurityOptions(c *testing.T) {
-	testRequires(c, testEnv.IsLocalDaemon, seccompEnabled, Apparmor, DaemonIsLinux)
+	testRequires(c, testEnv.IsLocalDaemon, DaemonIsLinux)
+	if !seccompEnabled() && !Apparmor() {
```

Comment:
We need to find a new home for some of these (or perhaps we have something like this somewhere in the new `integration/` suite, or in testutils